extensions/wikia/AdEngine/js/AdEngineRunner.js

```diff
@@ -6,7 +6,7 @@ define('ext.wikia.adEngine.adEngineRunner', [
 	'wikia.log',
 	'wikia.window',
 	require.optional('ext.wikia.adEngine.lookup.amazonMatch'),
-	require.optional('ext.wikia.adEngine.lookup.rubiconFastlane'),
+	require.optional('ext.wikia.adEngine.lookup.rubicon.rubiconFastlane'),
 	require.optional('ext.wikia.aRecoveryEngine.recovery.sourcePointRecovery')
 ], function (adEngine, adTracker, instantGlobals, log, win, amazonMatch, rubiconFastlane, spRecovery) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has 8 parameters which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AExcessiveParameterList)